tests/cli.rs

```diff
@@ -1329,6 +1330,25 @@ fn can_set_unset_header() {
         "#});
 }
 
+macro_rules! cookie_exists {
+    ($when:ident, $expected_value:expr) => {{
+        $when.matches(|req: &HttpMockRequest| {
+            req.headers
+                .as_ref()
+                .unwrap()
+                .iter()
+                .find(|(key, actual_value)| {
+                    key == "cookie" && {
+                        let expected = $expected_value.split("; ").collect::<HashSet<_>>();
+                        let actual = actual_value.split("; ").collect::<HashSet<_>>();
+                        actual == expected
+                    }
+                })
+                .is_some()
```

Comment:
You can use [`any()`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.any) instead